README.rst

```diff
@@ -22,7 +22,7 @@ Latest Release
   +-----------------------------------------------------------------------------+-------------------------------------------------------------------------------+
   | Versions                                                                    | Downloads                                                                     |
   +=============================================================================+===============================================================================+
-  | .. image:: https://img.shields.io/pypi/v/scikit-build.svg                   | .. image:: https://img.shields.io/badge/downloads-660k%20total-green.svg      |
+  | .. image:: https://img.shields.io/pypi/v/scikit-build.svg                   | .. image:: https://img.shields.io/pypi/dm/scikit-build                        |
```

Comment:
This seems simpler, and easier to keep up-to-date - we are getting ![downloads per month](https://img.shields.io/pypi/dm/scikit-build).